src/commands/synthetics/__tests__/run-test.test.ts

```diff
@@ -324,19 +324,23 @@ describe('run-test', () => {
         apiKey: 'fake_api_key',
         appKey: 'fake_app_key',
         configPath: 'fake-datadog-ci.json',
+        datadogSite: 'datad0g.com',
```

Comment:
```suggestion
        datadogSite: 'datadoghq.com',
```

Reply:
`datadoghq.com` is the default value in `DEFAULT_COMMAND_CONFIG`. I wanted to test that we correctly override this value

Reply:
Then could you use the `.eu` one? 🙏